aap-common/assembly-release-notes.adoc

```diff
@@ -0,0 +1,48 @@
+ifdef::context[:parent-context: {context}]
+
+[id="ap-release-notes"]
+
+= Release notes
+
+[discrete]
+== What’s new in Ansible Automation Platform?
+
+Check out the newest features and enhancements we've just released.
+
+This release includes a number of enhancements, additions, and fixes that have been implemented in the {PlatformNameShort}.
+
+[discrete]
+== New features and enhancements
+
+These releases of {AAPonAWS} and {AAPonGCP} of version 2.4.20230630-00 include the following enhancements:
+
+* {PlatformNameShort} 2.4 support
+** Automation Mesh and Event Driven Automation are not yet supported via the self-managed Ansible on Cloud offerings at this time
+* Added internal encryption to the web servers
+** Custom certificates can be added
+* Custom tagging/labelling support
+** AWS add/remove tag support for resources owned by the deployment
+** GCP add/remove label support for resources owned by the the deployment
+* {AAPonGCP} now has add/remove extension node operational playbooks
+* Backup and restore enhancements
+** Added support for taking multiple backups
+** Added operational playbook to list available backups
+** Added operational playbook to delete a selected backup
+** Added existing VPC restore capability
+* Added operational functionality to check current version
+** Operational playbooks now check to ensure that the {PlatformNameShort} environment they are operating upon is at the same version before beginning their operation.
+
+[discrete]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in {PlatformNameShort} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+The following is a list of major functionality deprecated and removed within {PlatformNameShort} 2.3:
+
+* On-premise component Automation Services Catalog is now removed from {PlatformNameShort} 2.4 onwards.
+* With the {PlatformNameShort} 2.4 release, the Execution Environment container image for Ansible 2.9 (ee-29-rhel-8) is no longer loaded into the Automation Controller configuration by default.
+
+[discrete]
+== Additional Release Notes related to Ansible Automation Platform
```

Comment:
I don't understand this section. Would it not be better to say:

For information about changes to {PlatformNameShort}, refer to the {PlatformNameShort} link:<xref to the AAP release notes>[Release Notes].